File: app/src/test/java/com/srikate/unittestexample/HelloTest.java
package com.srikate.unittestexample;

import com.srikate.unittestexample.basic.Hello;

import org.junit.Test;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertNotNull;

public class HelloTest {

    @Test
    public void shouldNotNull(){
        Hello hello = new Hello();
        assertNotNull(hello);
    }

    @Test
    public void shouldSayHello() {
        Hello hello = new Hello();
        String actualResult = hello.say();
        assertEquals("Hello", actualResult);
    }

}

File: app/src/test/java/com/srikate/unittestexample/HelloTest.kt
package com.srikate.unittestexample

import com.srikate.unittestexample.basic.Hello
import org.junit.Assert
import org.junit.Test

class HelloTest {
    @Test
    fun shouldNotNull() {
        val hello = Hello()
        Assert.assertNotNull(hello)
    }

    @Test
    fun shouldSayHello() {
        val hello = Hello()
        val actualResult = hello.say()
        Assert.assertEquals("Hello", actualResult)
    }
}